cli/factory.rs

```diff
@@ -488,7 +488,7 @@ impl CliFactory {
         async {
           Ok(Arc::new(CliGraphResolver::new(CliGraphResolverOptions {
             sloppy_imports_resolver: if self.options.unstable_sloppy_imports() {
-              Some(SloppyImportsResolver::new(self.fs().clone()))
+              Some(SloppyImportsResolver::new(self.fs().clone(), true))
```

Comment:
Reading this code, it's not clear what `true` means. Perhaps we should use a descriptive enum instead?